public/app/plugins/panel/graph/tab_display.html

```diff
@@ -9,7 +9,7 @@ <h5>Migration</h5>
         Migrate
       </button>
     </p>
-    <p>Some features like colored time regions and negative transforms are not supported in the new panel yet.</p>    
+    <p>Some features like negative transforms are not supported in the new panel yet.</p>
```

Comment:
this is outdated. we support negative transforms in TimeSeries field overrides.

Reply:
oh, didn't know that. Is there anything left that's not supported?